enterprise_access/apps/subsidy_access_policy/models.py

```diff
@@ -11,45 +11,83 @@
 from enterprise_access.apps.subsidy_access_policy.mocks import catalog_client, group_client, subsidy_client
 
 
+class PolicyManager(models.Manager):
+    def get_queryset(self):
+        return super().get_queryset().filter(policy_name=self.model.__name__)
+
+
 class SubsidyAccessPolicy(TimeStampedModel):
     """
     Tie together information used to control access to a subsidy.
-    This abstract model joins group, catalog, and access method.
+    This model joins group, catalog, and access method.
 
     .. no_pii:
     """
-    class Meta:
-        """
-        Metaclass for SubsidyAccessPolicy.
-        """
-        abstract = True
+
+    POLICY_FIELD_NAME = 'policy_name'
+    policy_name = models.CharField(max_length=64)
```

Comment:
nit: shouldn't it be called `policy_type` according to the diagram here https://2u-internal.atlassian.net/wiki/spaces/SOL/pages/300580900/EMET+Scrum+of+Scrums+Notes